bin/pocs_shell

```diff
@@ -152,7 +153,8 @@ class PocsShell(Cmd):
             simulator = []
 
         try:
-            observatory = Observatory(simulator=simulator)
+            cameras = create_cameras_from_config()
+            observatory = Observatory(simulator=simulator, cameras=cameras)
```

Comment:
FWIW, while thinking about adding a supervisor and better messaging, and also while reviewing Anthony's distributed cameras PR (which includes a nameserver), I wondered about introducing some kind of device registry. Something we can discuss in the future.

Reply:
Yes, I think we will definitely want to change how the Observatory interacts with and discovers devices. Hopefully this gets us closer to doing that.